### DISCUSSION TITLE
Webhook server

### DISCUSSION THREAD

**Discussion by jin-ahn** [actions/actions-runner-controller#1073] [Questions]
The documentation says that the only way to deploy a webhook server is through helm. I am wondering if this is still true or if the recent updates allows for webhook server to deployed by adding options to the yaml file? I want to avoid having to setup helm if possible on my eks cluster.

**Comment by vFondevilla** [comment]
You can render a YAML from the Helm chart using helm template (https://helm.sh/docs/helm/helm_template/)
